Upgrade thrift to 0.9.3

fishercoder1534 opened the pull request:
Upgrade Thrift to the latest version today: 0.9.3

Compiled and ran the test, all worked fine.

cakoose commented:
I noticed you updated the generated code in "tpc/pregen/media.thrift", which is the correct thing to update.  I also notice some additional generated files at "tpc/gen-java" -- what's that for?

Also, can you squash everything into a single commit?

fishercoder1534 commented:
Thanks a lot @cakoose for your advice, I've deleted my old fork and forked a new one and squashed everything into one single commit, also thanks for spotting those unnecessary files under "tpc/gen-java", I've cleaned them up. I'll close this Pull Request and open a new one for the same regard. Thanks again! :)
